Puppetfile

```diff
@@ -39,19 +39,30 @@ mod "puppetlabs/catalog_preview", :latest
 mod "puppet/archive", :latest
 mod "puppetlabs/inifile", :latest
 
-# Used by profile::puppet::pos_server
+# Used by profile::puppet::foss_server
 mod 'puppetlabs/postgresql', :latest
 mod 'puppetlabs/puppetdb', :latest
 mod 'puppet/puppetserver',
   :git => "https://github.com/voxpupuli/puppet-puppetserver.git",
   :tag => '2.1.0'
 mod 'puppetlabs/puppetserver_gem', :latest
 mod 'puppet/r10k', :latest
+mod 'puppetlabs/hocon', :latest
 # mod 'puppet/puppetboard', :latest
 
 # Used by profile::puppet::pe_code_manager
 mod "pltraining/rbac", '0.0.5'
 
+# Used by grafanadash server (puppet metrics)
+mod 'grafanadash',
+  :git => 'https://github.com/tuxmea/puppet-grafanadash.git',
+  :branch => 'master'
+mod 'dwerder/graphite', :latest
+mod 'stahnma/epel', :latest
+mod 'elastic-elasticsearch', :latest
+mod 'puppetlabs-java', :latest
```

Comment:
Same here,we have tinydata for jre and jdk https://github.com/example42/tinydata/tree/master/data/openjdk-jdk https://github.com/example42/tinydata/tree/master/data/openjdk-jre